### PR TITLE
ORD Aggregation of C4C CNS fails with local tenant id is required

### PR DESCRIPTION
**Pull request by dragobt**
**Description**

Changes proposed in this pull request:
- send tenant header only when webhook is associated with app template
- update dependencies

**Related issue(s)**

- ...

**Pull Request status**

- [x] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -139,7 +139,7 @@ global:
       name: compass-pairing-adapter
     director:
       dir:
-      version: "PR-2868"
```
Comment (la4ezar) [inline chart/compass/values.yaml:142]: Update the versions of all other components, because you've changed the go.sum/mod files.

**Review by la4ezar — CHANGES_REQUESTED** [review]

**Review by la4ezar — DISMISSED** [review]

**Review by alextargov — APPROVED** [review]

**Review by la4ezar — APPROVED** [review]

**Pull request merged**